lib/workers/global/limits.ts

```diff
@@ -16,14 +16,13 @@ export function resetAllLimits(): void {
 }
 
 export function setMaxLimit(key: Limit, max: unknown): void {
-  const maxVal = typeof max === 'number' && max > 0 ? max : null;
-  logger.debug(`${key} limit = ${maxVal}`);
-  const limit = limits.get(key);
-  limits.set(key, {
-    current: 0,
-    ...limit,
-    max: maxVal,
-  });
+  if (max === null || typeof max === 'number') {
```

Comment:
What's the difference here - `undefined` is skipped altogether?

Reply:
Never mind, `undefined` now also means limit reset